### DISCUSSION TITLE
Doubt about checking predictions with HoltWinters

### DISCUSSION THREAD

**Discussion by XaviMartiCamarasa** [online-ml/river#1094] [Q&A] [answered]
Hi . Im trying to forecast and getting the errors to plot them.But I have a doubt.
While running this code 

```
`from river import time_series, evaluate
from river import datasets
from river import metrics
dataset = datasets.AirlinePassengers()
metric = time_series.iter_evaluate(
     dataset=datasets.AirlinePassengers(),
     model=time_series.HoltWinters(alpha=0.1),
    metric=metrics.MAE(),
    horizon=12)

for m in metric:
      print(m)`
```

Why it is starting in 1950 and not 1949? as shown below?
![image](https://user-images.githubusercontent.com/117338671/203606910-6102eb8b-c7cc-40d3-b137-126808357271.png)

Many thanks!

**Comment by MaxHalford** [comment, marked as answer]
Hey there. No predictions are made for the first 12 samples, because there's not enough data before that to make a prediction. I agree that this may seem surprising, and we could add a parameter to try to do predictions anyway, or use a constant. But I think it's not bad like this actually. It's quite normal to want to have at least 12 months of data if you're predicting 12 months ahead. Agreed?

**Reply by XaviMartiCamarasa** [reply]
Fully agreed fot the 12 first ones, but....what about 1960? (12 last ones)

![image](https://user-images.githubusercontent.com/117338671/203628722-81b00ba0-7c76-4ef2-be5e-3aa16bfd4a25.png)
Last 12 month months in the data set:

![image](https://user-images.githubusercontent.com/117338671/203628808-a4dffea7-dafa-4fab-95c9-a0705e0ef848.png)

**Reply by MaxHalford** [reply]
Well likewise, the 12 last months in the dataset are the last ones the model can be evaluated on. You could predict values after that, but there would be no values to compare those predictions with.
